src/pages/Organization/AdminLogin.tsx

```diff
@@ -35,53 +40,53 @@ function AdminLogin() {
   };
 
   return (
-    <div className="md:flex md:flex-col md:items-center md:justify-center w-full  grow  text-center py-2  dark:bg-dark-bg bg-gray-100  sm:flex sm:flex-row sm:items-center sm:justify-center">
-      <div className="md:rounded-2xl md:shadow-2xl md:flex md:w-2/3 mt-16 md:max-w-4xl sm:max-w-xl sm:rounded-none sm:shadow-none dark:shadow-2xl mb-8">
-        <div className="md:w-3/5 md:p-5 sm:w-full sm:p-2 dark:bg-dark-frame-bg  dark:rounded-none">
+    <div className="w-full py-2 text-center bg-gray-100 md:flex md:flex-col md:items-center md:justify-center grow dark:bg-dark-bg sm:flex sm:flex-row sm:items-center sm:justify-center">
+      <div className="mt-16 mb-8 md:rounded-2xl md:shadow-2xl md:flex md:w-2/3 md:max-w-4xl sm:max-w-xl sm:rounded-none sm:shadow-none dark:shadow-2xl">
+        <div className="md:w-3/5 md:p-5 sm:w-full sm:p-2 dark:bg-dark-frame-bg dark:rounded-none">
           <div className="py-10 sm:py-8 ">
             <h2 className="text-2xl font-bold text-primary dark:text-dark-text-fill ">
               {t('Sign in using')}
             </h2>
-            <div className="border-2 w-10 bg-primary border-primary inline-block mb-2" />
+            <div className="inline-block w-10 mb-2 border-2 bg-primary border-primary" />
             <div className="flex justify-center my-2">
               <a
                 href="#link"
-                className="border-2 border-gray-200 rounded-full p-3 mx-1"
+                className="p-3 mx-1 border-2 border-gray-200 rounded-full"
               >
                 <FaGoogle className="text-sm dark:text-white" />
               </a>
             </div>
-            <p className="text-gray-400 my-3 dark:text-dark-text-fill ">
+            <p className="my-3 text-gray-400 dark:text-dark-text-fill ">
               {t('or use your email account')}
             </p>
             <div className="flex flex-col items-center">
               <form action="#none" onSubmit={handleSubmit(onSubmit)}>
-                <div className="bg-gray-100 w-64 p-2 flex items-center mb-2 dark:bg-dark-bg ">
-                  <FaRegEnvelope className="text-gray-400 mr-2" />
+                <div className="flex items-center w-64 p-2 mb-2 bg-gray-100 dark:bg-dark-bg ">
+                  <FaRegEnvelope className="mr-2 text-gray-400" />
                   <input
                     type="email"
                     {...register('email', { required: 'Email is required' })}
```

Comment:
The system should only allow work emails. These exclude regular domains like gmail, yahoo, outlook, etc